packages/celest/example/celest/project.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:celest/celest.dart';
+
+const project = Project(
+  name: 'hello',
```

Comment:
I feel like the name of the project and the app should be something like "my_flutter_project". Hello is confusing me. That should change both on the frontend and backend.